docs/.project/ALPHA_RELEASE.md

```diff
@@ -42,18 +43,16 @@ _To be done after code freeze and prior to the release day to prepare and build
     * [ ] Works on Mac
     * [ ] Works on Windows
 * [ ] trigger QA for fuzzy testing
-* [ ] write blog post for [blog.camunda.org](https://camunda.com/blog/) according to [this process](https://confluence.camunda.com/pages/viewpage.action?spaceKey=MAR&title=Your+Content+on+the+Camunda+Blog)
-   * [ ] copy and paste [this template](https://docs.google.com/document/d/18aXWFBdbET97EZ_HO1DEVLVsYP-UQEabumSzQdeRbDw/edit?usp=sharing) to a new docs file in [Modeling blog posts](https://drive.google.com/drive/u/1/folders/1Kw3lo2zbl3HL7w6y19A3S9SdpAdrDcFJ)
-   * [ ] write a draft version of the blog post
-   * [ ] send it to Charley Mann and Christopher Rogers for review and ask them to coordinate the publishing
+* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/)
+   * [ ] create a draft post on Wordpress
+   * [ ] send the draft to Charley Mann for review
+   * [ ] send the reviewed draft to Christopher Rogers to coordinate the publishing
 
 _To be done on release day to announce the release and making it publically available._
 
 * [ ] publish release on [Github Releases](https://github.com/camunda/camunda-modeler/releases)
 * [ ] trigger mirroring of release to https://camunda.org/release/camunda-modeler/ via [Jenkins](https://ci.cambpm.camunda.cloud/job/sideprojects/job/camunda-modeler-desktop-RELEASE/build?delay=0sec)
-* [ ] spread the word
-    * [ ] send [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email)
-    * [ ] tweet
+* [ ] send the [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/camunda-modeler/README.md#release-notice-email) to `alle`
```

Comment:
If I understand https://github.com/bpmn-io/internal-docs/issues/180 correctly, we should *not* send the email to `alle`.

Reply:
Check the _Applies To_ section, it excludes alpha releases.

Reply:
All right!